auth-web/src/resources/BreadcrumbResources.ts

```diff
@@ -0,0 +1,19 @@
+import { BreadcrumbIF } from '@bcrs-shared-components/interfaces'
```

Comment:
Is this common for all applications? if so we can import from shared components only. We don't have a resources folder yet. Can we use this file inside the util folder for better structure? This will be one file inside folder

Reply:
This isn't common for all as of yet. 
I believe once the Uber Dashboard or highest level landing/marketing page is complete, we can then then create a common resource. Could look at doing this now though, whatever you prefer.

Reply:
To follow up on this, it will be tricky as until each UI has a set configuration object that differs between them. 
ie the filings UI refers to Auth Web home as the `BUSINESS_URL` and auth web refers to itself as `AUTH_WEB_URL` etc. 
So in order to make 1 common resource we would either need..
a) all the configuration objects to follow a standard naming convention
b) have local customization of these resourced objects (which we sort of land us in the same spot we are now anywho)